examples/gpx.html

```diff
@@ -46,15 +43,64 @@
             itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
             itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
 
+            // update the waypoint
+            const positionCamera = new itowns.THREE.Vector3();
+            function getDistance(object, camera) {
+                const point = object.geometry.boundingSphere.center.clone().applyMatrix4(object.matrixWorld);
+                positionCamera.setFromMatrixPosition(camera.matrixWorld);
```

Comment:
you could use `camera.position` directly.

Reply:
Done